WcaOnRails/db/migrate/20210721135845_add_free_entry_text_to_competitions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddFreeEntryTextToCompetitions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :Competitions, :free_entry_text, :string, null: true, default: "none"
```

Comment:
Why allow `NULL` values if you provide a default anyways?

Reply:
oops